packages/modeling/src/geometries/geom2/toSides.js

```diff
@@ -11,6 +11,15 @@ import applyTransforms from './applyTransforms.js'
  * @example
  * let sharedsides = toSides(geometry)
  */
-export const toSides = (geometry) => applyTransforms(geometry).sides
+export const toSides = (geometry) => {
```

Comment:
I think that applyTransforms should be called before constructing the sides. Every to* function should be calling applyTransforms() to make sure the outlines(points) are ready for export.

Reply:
The `toSides` function calls `toOutlines`, and that function calls `applyTransforms`. So this is already handled.